services/web/server/src/simcore_service_webserver/projects/_create_utils.py

```diff
@@ -151,7 +152,7 @@ async def _compose_project_data(
 
 async def create_project(
     task_progress: TaskProgress,
-    app: web.Application,
+    request: web.Request,
```

Comment:
do you really need to pass the request here? I had the feeling you finally only need the application. Using the request does restrict when you can use the method.